bench/gemm-benchmark.h

```diff
@@ -402,7 +402,7 @@ void GEMMBenchmark(benchmark::State& state,
 
   const size_t mc = state.range(0);
   const size_t nc = state.range(1);
-  const size_t kc = state.range(2);
+  const size_t kc = round_up_po2(state.range(2), 2 * kr * sr);
```

Comment:
kc does not require alignment?

Reply:
This is to satisfy the constraints kernels have for kc % bl == 0 i.e. padding for benchmark to support various shapes. Not sure if I follow your comment about alignment.